### PR TITLE
Use get_n2 helper function

### PR DESCRIPTION
**Pull request by atdotde**
Now, that we have this helper function that should have been
introduced long ago, we can make some more expressions
more idiomatic.

Signed-off-by: Robert C. Helling <helling@atdotde.de>

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [x] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### CODE REVIEW

File: core/gas.c [hunk 1]
```diff
@@ -16,7 +16,7 @@ bool isobaric_counterdiffusion(struct gasmix oldgasmix, struct gasmix newgasmix,
 {
 	if (!prefs.show_icd)
 		return false;
-	results->dN2 = get_he(oldgasmix) + get_o2(oldgasmix) - get_he(newgasmix) - get_o2(newgasmix);
+	results->dN2 = get_n2(oldgasmix) - get_n2(newgasmix);
```
Comment (bstoeger) [inline core/gas.c]: Doesn't that invert the sign of dN2? I.e. shouldn't that be
```
	results->dN2 = get_n2(newgasmix) - get_n2(oldgasmix);
```
?

**Comment by bstoeger** [comment]
Looks good to me, up to one comment.

File: core/gas.c (hunk 1 above)
Comment (atdotde) [reply core/gas.c]: Oops.

**Comment by bstoeger** [comment]
Looks good - thanks (one might argue that "inert = 1000 - o2" is safer with respect to adding other "intert" gases such as H2, but I reckon that this is the slightest of the problems).

**Pull request merged**